.github/workflows/chart-publish.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     permissions:
       contents: read
       packages: write
+      id-token: write # To sign the provenance.
```

Comment:
we should test if this needs to be write, but the sigstore example workflow says it is.